Libraries/JavaScriptAppEngine/Initialization/InitializeJavaScriptAppEngine.js

```diff
@@ -44,6 +44,22 @@ function handleError(e, isFatal) {
   }
 }
 
+/**
+ * Assigns a new global property, replacing the existing one if there is one.
+ * 
+ * Existing properties are preserved as `originalPropertyName`. Both properties
+ * will maintain the same enumerability & configurability.
+ */
```

Comment:
ok, sorry for the change but can you add an example of why we are doing that. Telling people to do in their app if they want to use the default XHR to get Chrome/Safari/Firefox in devtools if they can manage CORS themselves would be great.

```
GLOBAL.XMLHTTPRequest = GLOBAL.originalXMLHTTPRequest;
```

Otherwise this is likely going to be either removed or stay there forever and people are going to be afraid of touching it because it's doing some black magic